TravelBug/src/WebPages/loginPage/LoginPage.jsx

```diff
@@ -0,0 +1,102 @@
+import "./Loginpage.css";
+import { useState } from "react";
+import { useAuth } from "../../Contexts/AuthContext";
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { useData } from "../../Contexts/Datacontext";
+
+const LoginPage = () => {
+  const navigate = useNavigate();
+  const { loginCall, token, user } = useAuth();
+  const { dispatch } = useData();
+  const [loginDetails, setLoginDetails] = useState({ email: "", password: "" });
+
+  const guestLoginHandler = () => {
+    setLoginDetails({
+      email: "adarshbalika@gmail.com",
+      password: "adarshbalika",
+    });
+    loginCall("adarshbalika@gmail.com", "adarshbalika");
+  };
+
+  useEffect(() => {
+    if (token) {
+      setTimeout(() => {
+        navigate("/");
+      }, 1000);
```

Comment:
You can remove this settimeout as it will give a bad user impression

Reply:
yes thanks